aws/rust-runtime/aws-config/src/sso.rs

```diff
@@ -63,13 +59,9 @@ impl SsoCredentialsProvider {
         let fs = provider_config.fs();
         let env = provider_config.env();
 
-        let mut sso_config = SsoConfig::builder()
-            .http_connector(expect_connector(
-                "The SSO credentials provider",
-                provider_config.connector(&Default::default()),
-            ))
-            .retry_config(RetryConfig::standard());
-        sso_config.set_sleep_impl(provider_config.sleep());
+        let mut sso_config = SsoConfig::builder().retry_config(RetryConfig::standard());
+        sso_config.set_http_client(provider_config.http_client());
```

Comment:
does this need the `if let Some(...)` treatment?

Reply:
This code was removed after merging the provider config refactor in.